### PR TITLE
Bundle OpenJCEPlus on Mac/Linux

### PR DESCRIPTION
**Pull request by JinhangZhang**
OpenJCEPlus providers are supported on the Mac OS with aarch64 and amd64 architectures. Also on the Linux aarch64. This PR aims to add aarch64-mac, x86_64-mac and Linux aarch64 to platforms bundling OpenJCEPlus.

Backport from https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/888 & https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/896

### CODE REVIEW

File: closed/make/copy/Copy-openjceplus.gmk [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# ===========================================================================
+ # ===========================================================================
```
Comment (keithc-ca) [inline closed/make/copy/Copy-openjceplus.gmk:1]: Please revert this change.

**Review by keithc-ca — CHANGES_REQUESTED** [review]

File: closed/make/copy/Copy-openjceplus.gmk (hunk 1 above)
Comment (JinhangZhang) [reply closed/make/copy/Copy-openjceplus.gmk]: updated

**Comment by keithc-ca** [comment]
Jenkins compile alinux,amac,osx jdk11

**Review by keithc-ca — APPROVED** [review]
